main.go

```diff
@@ -18,18 +39,69 @@ func getPing(w http.ResponseWriter, r *http.Request) {
 
 }
 
-func getBooks(w http.ResponseWriter, r *http.Request) {
+func Books(w http.ResponseWriter, r *http.Request) {
+	metodo := r.Method
+	fmt.Printf("metodo: %v\n", metodo)
```

Comment:
I don't think we want this `fmt.Printf` here. We will discuss logging later.

Reply:
Okay. It was like a "flag" to see what was happening.